doc/versionHistory.rst

```diff
@@ -6,6 +6,13 @@
 Version History
 ##################
 
+.. _lsst.ts.ofc-3.2.1:
+
+v3.2.1
+======
+
+* Add comcam to policy/sensor_ids_to_names.yaml.
```

Comment:
Since we have not released this version yet, maybe we can include this in version 3.2.0?